msteams-platform/graph-api/rsc/preapproval-instruction-docs.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Add, update, and remove pre-approvals for RSC enabled applications in Teams
+description: In this article, learn more about creating, managing, and deleting pre-approvals for RSC enabled applications in MS Teams
+ms.localizationpriority: medium
+author: MSFTRickyCastaneda
+ms.author: rcastaneda
+ms.topic: Conceptual
+ms.date: 08/29/2023
+---
+
+In this article:
+1. [What is preapproval of RSC permissions](#what-is-preapproval-of-rsc-permissions)
+1. [Setting up PowerShell to manage preapprovals](#setting-up-powershell-to-manage-preapprovals)
+1. [Enable pre-approval for RSC permissions](#enable-preapproval-for-rsc-permissions)
+1. [Create a preapproval](#create-a-preapproval)
+      * [With a sensitivity label](#create-a-preapproval-with-a-sensitivity-label)
+      * [Without a sensitivity label](#create-a-preapproval-without-a-sensitivity-label)
+1. [Modify a preapproval](#modify-a-preapproval)
+1. [Delete a preapproval](#delete-a-preapproval)
+
+# What is preapproval of RSC permissions?
+
+Preapproval of RSC permissions allows admins to approve resource-specific consent (RSC) on an app-by-app basis in Microsoft Teams. Previously, admins had the ability to toggle RSC on or off in their tenant. Admins had to approve or block all RSC enabled apps with no granularity to declare which apps they wanted to allow. Through pre-approved RSC, admins can choose which RSC-enabled apps they allow their tenant users to install in Teams without having to allow or block all apps that utilize RSC permissions.
```

Comment:
RSC preapproval provides admins with granular control over the resource-specific consent (RSC) permissions an app can request upon installation. Today, RSC permissions are granted to an app only when the app gets installed. Through the use of pre-approval policies, admins can declare ahead of time the max permissions an app can request from the end user and be consented to during installation. For a permission pre-approval policy to take affect for an app, the admin still needs to ensure that the app is enabled (installable) in their organization (link to managing apps in TAC). If the app is not installable, then the permission pre-approval with respect to that specific app is moot.

Admins have the ability to create finely tuned pre-approval policies based on dimensions such as app id, permissions and sensitivity of the data being accessed. Pre-approval for RSC permissions is an advanced feature meant for admins who have a need to creating advanced custom policies for their organization. Out of the box, the preapproval policies are managed by Microsoft, and it is recommended that organizations leave it in that state. This allows the security team at Microsoft to provide the best security posture for your organization.

Reply:
Please rephrase for clarity.